src/matlab2tikz.m

```diff
@@ -3842,8 +3845,52 @@ function warnFacetedInterp(m2t)
     [m2t, drawOptions] = setColor(m2t, h, drawOptions, 'fill', faceColor);
 end
 % ==============================================================================
+function [m2t,str] = drawBaseline(m2t,hparent,isVertical)
+    str = '';
+    baseValue = get(hparent, 'BaseValue');
+    if isOff(get(hparent,'ShowBaseLine')) || ~isHG2() || baseValue ~= 0
+        return
+    end
+    
+    if ~exist('isVertical','var')
+        isVertical = false;
+    end
+
+    hBaseLine = get(hparent,'BaseLine');
+
+    % Line options of the baseline
+    lineStyle        = get(hBaseLine, 'LineStyle');
+    lineWidth        = get(hBaseLine, 'LineWidth');
+    lineOptions      = getLineOptions(m2t, lineStyle, lineWidth);
+    color            = get(hBaseLine, 'Color');
+    [m2t, lineColor] = getColor(m2t, hBaseLine, color, 'patch');
+
+    drawOptions = opts_new();
+    drawOptions = opts_add(drawOptions, 'forget plot');
+    drawOptions = opts_add(drawOptions, 'color', lineColor);
+    drawOptions = opts_merge(drawOptions, lineOptions);
+    drawOpts    = opts_print(m2t, drawOptions, ',');
+
+    % Get data
+    if isVertical
+        xData = repmat(baseValue,1,2);
+        yData = get(m2t.currentHandles.gca,'Ylim');
+    else
+        xData = get(m2t.currentHandles.gca,'Xlim');
+        yData = repmat(baseValue,1,2);
+    end
+    
+    [m2t, table, tabOpts] = makeTable(m2t, '', xData, '', yData);
+
+    str = sprintf('%s\\addplot[%s] plot table[%s] {%s};\n', ...
```

Comment:
Shouldn't this be `%s\\addplot[%s] table[%s] {%s};\n` ? I know we have the `plot table` somewhere else in the code as well (and it compiles just fine), but in the `pgfplots` manual I can only find `\addplot table`, never `\addplot plot table`, though `plot table` is used as a search term in the manual.

What do you think?